src/components/DataTable.js

```diff
@@ -7,12 +7,25 @@ export default class DataTable extends React.Component {
     this.state = {
       page: 0,
       wide: false,
+      sort: {
+        column: '',
+        order: 'asc',
+      },
     };
     this.resize = this.resize.bind(this);
     this.prevPage = this.prevPage.bind(this);
     this.nextPage = this.nextPage.bind(this);
   }
 
+  componentWillMount() {
+    this.setState({
```

Comment:
Yeah, exactly. I tried to put it in the constructor, but `this.props.headers` wasn't defined yet.